From 39a87a1ac0e252286ec23a4aed80f8e0a89db95c Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Sat, 16 Dec 2017 22:23:19 +0200
Subject: [PATCH 036/249] wiki: Update r5/r7plus device info

* Add me as r7plus co-maintainer
* Add missing items, align with other devices
* Provide custom TWRP that supports:
  - Both Spectrum and ColorOS firmware compatibility for r7plus
  - Hardware crypto decryption for both devices

Change-Id: Iafae9416518f27eaab4d10f503febb9bde09ffe4
---
 _data/devices/r5.yml     | 25 +++++++++++++++----------
 _data/devices/r7plus.yml | 33 +++++++++++++++++----------------
 2 files changed, 32 insertions(+), 26 deletions(-)

diff --git a/_data/devices/r5.yml b/_data/devices/r5.yml
index c880da7..af3b524 100644
--- a/_data/devices/r5.yml
+++ b/_data/devices/r5.yml
@@ -1,5 +1,5 @@
 architecture: arm
-battery: 2000 mAh (Non-removable)
+battery: Non-removable Li-Po 2000 mAh
 bluetooth: '4.0'
 cameras:
 - {flash: 'LED', info: '13 MP'}
@@ -9,38 +9,43 @@ channels: [weekly]
 codename: r5
 cpu: Cortex A53
 cpu_cores: '8'
-cpu_freq: 1.5 GHz
+cpu_freq: 1.5 GHz and 1.0 GHz
 current_branch: 14.1
+custom_twrp_link: https://androidfilehost.com/?fid=889964283620772636
+depth: 4.9 mm (0.19 in)
 download_boot: With the device powered off, hold <kbd>Volume Up</kbd> + <kbd>Power</kbd>.
 gpu: Adreno 405
 has_recovery_partition: true
+height: 148.9 mm (5.86 in)
 image: r5.png
 install_method: fastboot_oppo
 kernel: android_kernel_oppo_msm8939
 maintainers: [maniac103]
+models: [R8106, R8106s]
 name: R5/R5s (International)
 network:
-- {tech: 2G, bands: '' }
-- {tech: 3G, bands: '' }
-- {tech: 4G, bands: '' }
+- {tech: 2G, bands: '850 900 1800 1900 MHz GSM' }
+- {tech: 3G, bands: '850 900 1700 1900 2100 MHz HSPA+ (R5), 850 900 1900 2100 MHz HSPA+ (R5s)' }
+- {tech: 4G, bands: '1(2100) 3(1800) 4(1700/2100) 7(2600) 8(900) 17(700) 20(800) 28(700) 40(2300) LTE(Cat. 4) (R5), 1(2100) 3(1800) 7(2600) 40(2300) LTE(Cat. 4) (R5s)' }
 no_oem_unlock_switch: true
-peripherals: ['']
+peripherals: [A-GPS, Accelerometer, Compass, GLONASS, GPS, Gyroscope, Light sensor, Magnetic sensor, Proximity sensor, USB OTG]
 project_spectrum: "https://androidfilehost.com/?fid=962021903579484169"
 project_spectrum_recovery: "https://androidfilehost.com/?fid=889964283620757488"
-ram: 2 GB
+ram: 2 GB (R5), 3 GB (R5s)
 recovery_boot: With the device powered off, hold <kbd>Volume Down</kbd> + <kbd>Power</kbd>.
-release: 2014 December
+release: 2014 December (R5), 2015 August (R5s)
 required_bootloader_link: "https://www.androidfilehost.com/?fid=24566382913912839"
 screen: 132 mm (5.2 in)
-screen_ppi: ~423
+screen_ppi: '~423'
 screen_res: 1080x1920
 screen_tech: AMOLED
 sdcard:
 soc: Qualcomm MSM8939 Snapdragon 615
-storage: 16 GB
+storage: 16 GB (R5), 32 GB (R5s)
 tree: android_device_oppo_r5
 type: phone
 vendor: OPPO
 vendor_short: oppo
 versions: [14.1]
+width: 74.5 mm (2.93 in)
 wifi: 802.11 a/b/g/n, Dual-band
diff --git a/_data/devices/r7plus.yml b/_data/devices/r7plus.yml
index 707e0e9..69151fb 100644
--- a/_data/devices/r7plus.yml
+++ b/_data/devices/r7plus.yml
@@ -1,9 +1,9 @@
 architecture: arm64
-battery: 4100 mAh Li-Po
+battery: Non-removable Li-Po 4100 mAh
 bluetooth: '4.0'
 cameras:
-- {flash: 'Dual LED', info: '13 MP Sony IMX278'}
-- {flash: '', info: '8 MP OmniVision OV8858'}
+- {flash: 'Dual LED', info: '13 MP'}
+- {flash: '', info: '8 MP'}
 carrier:
 channels: [weekly]
 codename: r7plus
@@ -11,23 +11,24 @@ cpu: Cortex A53
 cpu_cores: '8'
 cpu_freq: 1.5 GHz and 1.0 GHz
 current_branch: 14.1
-custom_twrp_codename: r7plusf
-depth:
+custom_twrp_link: https://androidfilehost.com/?fid=673791459329067727
+depth: 7.8 mm (0.31 in)
 download_boot: With the device powered off, hold <kbd>Volume Up</kbd> + <kbd>Power</kbd>.
 gpu: Adreno 405
 has_recovery_partition: true
-height:
+height: 158 mm (6.22 in)
 image: r7plus.png
 install_method: fastboot_nexus
 kernel: android_kernel_oppo_msm8939
-maintainers: [maniac103]
-name: R7 Plus
+maintainers: [maniac103, mikeioannina]
+models: [R7plusf]
+name: R7 Plus (International)
 network:
-- {tech: 2G, bands: '' }
-- {tech: 3G, bands: '' }
-- {tech: 4G, bands: '' }
-peripherals: [Dual SIM, Fingerprint reader]
-ram: 3 GB
+- {tech: 2G, bands: '850 900 1800 1900 MHz GSM' }
+- {tech: 3G, bands: '850 900 1900 2100 MHz HSPA+' }
+- {tech: 4G, bands: '1(2100) 3(1800) 5(850) 7(2600) 8(900) 40(2300) LTE(Cat. 4)' }
+peripherals: [A-GPS, Accelerometer, Compass, Dual SIM, GLONASS, GPS, Gyroscope, Light sensor, Magnetic sensor, Proximity sensor, USB OTG, Fingerprint reader]
+ram: 3/4 GB
 recovery_boot: With the device powered off, hold <kbd>Volume Down</kbd> + <kbd>Power</kbd>.
 release: 2015 May
 screen: 152 mm (6.0 in)
@@ -36,11 +37,11 @@ screen_res: 1080x1920
 screen_tech: AMOLED
 sdcard: up to 128 GB
 soc: Qualcomm MSM8939 Snapdragon 615
-storage: 32 GB
+storage: 32/64 GB
 tree: android_device_oppo_r7plus
 type: phone
 vendor: OPPO
 vendor_short: oppo
 versions: [14.1]
-width:
-wifi: 802.11 a/b/g/n/ac
+width: 82 mm (3.23 in)
+wifi: 802.11 a/b/g/n/ac, Dual-band
-- 
2.17.1

